### PR TITLE
Admin fixes

### PR DESCRIPTION
**Pull request by chrisrodz**
### Fixed a bunch of bugs in the admin.

I just completed all of the bugs / fixes on my pivotal board for the admin. Specifics below:
- [x] Fixed redirection bug in admin by removing trailing slashes from urls.
- [x] Added single model entry view that Ainsley will style later. (**If you want, don't merge until we style today**)
- [x] Fixed some labels and stuff that was on my pivotal board.

For reviewing later today @MrMaksimize  :smile:

### CODE REVIEW

**Comment by ainsleywagon** [comment]
Okay I think this is good to go.
screen shot 2014-07-15 at 6 18 43 pm

File: app.js [hunk 1]
```diff
@@ -130,6 +130,15 @@ app.use(function(req, res, next) {
   next();
 });
 
+// Middleware to remove trailing slashes from urls.
+app.use(function(req, res, next) {
```
Comment (MrMaksimize) [inline app.js:134]: Spacing

File: lib/options.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ var choicesList = {
   ],
   eligibleBusinessLocation: {
     'anywhere_in_pr': 'Anywhere In Puerto Rico',
-    'municipality_in_pr': 'Municipality in Puerto Rico',
+    'municipality_in_pr': 'Municipality in Puerto Rico (Please specify below)',
```
Comment (MrMaksimize) [inline lib/options.js]: Ummmmm... what's going on here?

File: public/src/js/main.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ $(document).ready(function() {
   $('div#eligibleIndustries').next().show();
```
Comment (MrMaksimize) [inline public/src/js/main.js:80]: If you have some time @chrisrodz would love this to be in its own CommonJS compatible file

File: views/admin/entry.jade [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        a.btn.btn-default(href="#{base}/#{model}", role="button") Back to all #{model}
+
+    table.table.table-striped.table-hover.table-condensed.admin
+      thead
+        tr
+          th
+            a.btn.btn-default.admin-edit.entry(href="#{base}#{path}/edit") edit
+            a.btn.btn-default.delete-model.admin-edit.entry(href="#{base}#{path}/delete") delete
+      tbody.admin
+
```
Comment (MrMaksimize) [inline views/admin/entry.jade]: I'd say no space here :)

**Comment by MrMaksimize** [comment]
Mostly minor stuff :)  Wanted to know what's up with that options change :)
Buenos noches!

File: lib/options.js (hunk 2 above)
Comment (chrisrodz) [inline lib/options.js]: Clara happened. It was part of the fixes tothe admins that she put on
pivotal. Basically use the other field to include the specific municipality.

On Tuesday, July 15, 2014, Maksim Pecherskiy notifications@github.com
wrote:

> In lib/options.js:
> 
> > @@ -46,7 +46,7 @@ var choicesList = {
> >    ],
> >    eligibleBusinessLocation: {
> >      'anywhere_in_pr': 'Anywhere In Puerto Rico',
> > -    'municipality_in_pr': 'Municipality in Puerto Rico',
> > -    'municipality_in_pr': 'Municipality in Puerto Rico (Please specify below)',
> 
> Ummmmm... what's going on here?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CoquiCoders/bizwallet/pull/34/files#r14978204.

## 

Best,
Christian A. Rodriguez

Comment (MrMaksimize) [inline lib/options.js]: Ok.... we need to talk about this.   This is not ok.  Most of this PR is
not related, so if you can just cut that small part out we can take it.
 I'm done for today.

This is a convo for tomorrow.

On Tue, Jul 15, 2014 at 6:56 PM, Christian A. Rodriguez <
notifications@github.com> wrote:

> In lib/options.js:
> 
> > @@ -46,7 +46,7 @@ var choicesList = {
> >    ],
> >    eligibleBusinessLocation: {
> >      'anywhere_in_pr': 'Anywhere In Puerto Rico',
> > -    'municipality_in_pr': 'Municipality in Puerto Rico',
> > -    'municipality_in_pr': 'Municipality in Puerto Rico (Please specify below)',
> 
> Clara happened. It was part of the fixes tothe admins that she put on
> pivotal. Basically use the other field to include the specific municipality.
> … <#1473ce3022fdaa93_>
> On Tuesday, July 15, 2014, Maksim Pecherskiy notifications@github.com
> wrote: In lib/options.js: > @@ -46,7 +46,7 @@ var choicesList = { > ], >
> eligibleBusinessLocation: { > 'anywhere_in_pr': 'Anywhere In Puerto Rico',
> 
> > - 'municipality_in_pr': 'Municipality in Puerto Rico', > +
> >   'municipality_in_pr': 'Municipality in Puerto Rico (Please specify below)',
> >   Ummmmm... what's going on here? — Reply to this email directly or view it
> >   on GitHub <
> >   https://github.com/CoquiCoders/bizwallet/pull/34/files#r14978204>.
> >   -- Best, Christian A. Rodriguez
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CoquiCoders/bizwallet/pull/34/files#r14978241.

## 

_Maksim Pecherskiy_
Code For America http://codeforamerica.org | _Puerto Rico (team blog
http://coquicoders.org)_
_773.677.7755 || Schedule A Meeting https://calendly.com/maksim/ *
*@MrMaksimize http://twitter.com/MrMaksimize || MrMaksimize.com
http://mrmaksimize.com_
</>

**Comment by chrisrodz** [comment]
Fixed the stuff. Removed the municipality thing. Will make a separate PR for that.

**Comment by MrMaksimize** [comment]
Me gusta lo que yo veo :)  :+1: @chrisrodz @ainsleywagon

**Pull request merged**